pkg/blackduck/ctl_blackduck.go

```diff
@@ -213,6 +228,8 @@ func (ctl *Ctl) AddSpecFlags(cmd *cobra.Command, master bool) {
 	cmd.Flags().StringVar(&ctl.AuthCustomCAFilePath, "auth-custom-ca-file-path", ctl.AuthCustomCAFilePath, "Absolute path to a file for the Custom Auth CA for Black Duck")
 	cmd.Flags().StringVar(&ctl.Type, "type", ctl.Type, "Type of Black Duck")
 	cmd.Flags().StringVar(&ctl.DesiredState, "desired-state", ctl.DesiredState, "Desired state of Black Duck")
+	cmd.Flags().BoolVar(&ctl.DbMigrateState, "state-db-migrate", ctl.DbMigrateState, "Create Black Duck in the database-migration state")
```

Comment:
How about --migration-mode (or maybe even just --migrate) as the flag on create?